homeassistant/components/apple_tv/remote.py

```diff
@@ -35,36 +65,34 @@ def name(self):
     @property
     def unique_id(self):
         """Return a unique ID."""
-        return self._atv.metadata.device_id
+        return f"remote_{self._identifier}"
```

Comment:
```suggestion
        return self._identifier
```

unique_id is automatically "prefixed" by the domain